components/media_player/ui/CardTVChannelList

```diff
@@ -0,0 +1,253 @@
+/******************************************************************************
+ *
+ * Copyright (C) 2018-2019 Marton Borzak <hello@martonborzak.com>
```

Comment:
Is this Marton's clone?

Reply:
this is not even a qml file.....It's always there but Idelet it again